changelogs/current.yaml

```diff
@@ -73,6 +73,12 @@ minor_behavior_changes:
 - area: admin
   change: |
     Switch no admin ``warning`` -> ``info``.
+- area: upstream
+  change: |
+    Fixed a reported issue (https://github.com/envoyproxy/envoy/issues/11004) that causes the Least
+    Request load balancer policy to be unfair when the number of hosts are very small, when the number
```

Comment:
Please break this sentence up, currently its not grammatically correct and the meaning is hard to decipher

Reply:
This will be "fixed" by removing the entry entirely. Since we have decided to make full scan mode always require opt in (even if choice count > # hosts), this is no longer even an accurate/applicable entry.